### ISSUE TITLE
Booleans and numbers are not supported in json models

### ISSUE THREAD

**Issue by Quenty** [LPGhatguy/rojo#30]
```
{
  "name": "InsertDefaultModules",
  "className": "BoolValue",
  "children": [ ],
  "properties": {
  	"Value": {
      "type": "boolean",
      "value": true
    }
  }
}
```

Expected behavior: It works

Actual behavior: errors

![image](https://user-images.githubusercontent.com/1774409/34847052-b6863794-f6df-11e7-906a-31b881839156.png)

**Comment by LPGhatguy** [comment]
Still need to implement the rest of the value types in the `RbxValue` enum, which will eventually include types like `Vector3` too!

**Comment by LPGhatguy** [comment]
Fixed by c5f824754!
